modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) access control list (ACL) to manage egress firewalls and network policies. Audit logging exposes `allow` and `deny` ACL events for `NetworkPolicy` and `BaselineAdminNetworkPolicy` custom resources (CR). Logging also exposes `allow`, `deny`, and `pass` ACL events for `AdminNetworkPolicy` (ANP) CR.
+
+You can configure the destination for audit logs, such as a syslog server or a UNIX domain socket.
+Regardless of any additional configuration, an audit log is always saved to `/var/log/ovn/acl-audit-log.log` on each OVN-Kubernetes pod in the cluster.
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+ annotations:
+  k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+ name: anp-tenant-log
+spec:
+ priority: 5
+ subject:
+  namespaces:
+   matchLabels:
+     kubernetes.io/metadata.name: tenant
+ egress:
+ - name: "deny-ns-tenant-1"
+  action: "Deny"
+  to:
+  - namespaces:
+    matchLabels:
```

Comment:
We need to fix indentation for matchLabels and the following key kubernetes.io/metadata.name